### PR TITLE
add new application: com.github.emmanueltouzery.hotwire

### PR DESCRIPTION
**Pull request by emmanueltouzery**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

There is a patch for the rust `relm` library, but this patch was already accepted by upstream. I also patch my own app's build dependencies, because I had trouble in vendoring that library dependency because I'm depending on a snapshot of the library. So the flatpak build depends on the latest stable version of the library, plus that patch, while the github version of my app depends on a git snapshot of the library. This will go away in the near future when I can depend on newer released versions of this dependency.

The app's home page is at https://github.com/emmanueltouzery/hotwire and I'm the main author.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, builb com.github.emmanueltouzery.hotwire

File: com.github.emmanueltouzery.hotwire.json [hunk 1]
```diff
@@ -0,0 +1,129 @@
+{
+  "app-id": "com.github.emmanueltouzery.hotwire",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "hotwire",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
```
Comment (hfiguiere) [inline com.github.emmanueltouzery.hotwire.json:10]:
```suggestion
```

Comment (hfiguiere) [reply com.github.emmanueltouzery.hotwire.json:10]: Usually if the app supports wayland, it doesn't have x11, just fallback-x11 (which is only available if there is no wayland available)

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by LeandroStanger** [comment]
bot, build com.github.emmanueltouzery.hotwire

**Pull request closed without merging**